precompiles/outposts/osmosis/tx_test.go

```diff
@@ -18,6 +18,10 @@ import (
 	"github.com/evmos/evmos/v15/utils"
 )
 
+const (
+	aevmosDenom = "aevmos"
```

Comment:
there's a `BaseDenom` constant in `/utils/utils.go` that you can use instead of declaring this constant